frontend/packages/console-shared/src/components/formik-fields/SelectInputField.tsx

```diff
@@ -41,7 +42,10 @@ const SelectInputField: React.FC<SelectInputFieldProps> = ({
   };
 
   const onCreateOption = (newVal: string) => {
-    setNewOptions([...newOptions, { value: newVal, disabled: false }]);
+    const duplicates = newOptions.filter((option) => option.value === newVal);
+    if (!preventDuplicates || duplicates.length === 0) {
```

Comment:
Wondering if we really need to introduce a prop here. Why not simply check if the newVal does not already exist in the list then add it to the list. Also, you would need to include the options coming from the `options` prop as in the new Val must not be present in the `options` & `newOptions`

cc @invincibleJai

Reply:
i agree i can't of a scenario where we would need duplicates 👍

Reply:
+1

Reply:
alright I have made the changes